src/clj/web/app_state.clj

```diff
@@ -10,6 +10,7 @@
   (atom {:lobbies {}
          :lobby-updates {}
          :tournament nil
+         :allow-game-creation true
```

Comment:
minor: As a general rule, I prefer defaults to begin false/empty because they are less surprising.  Here, `pause-game-creation` would invert this so that "false" is the desired state.  It inverts the logic check, but aligns the state to the button on the admin page, simplifying that.

Reply:
I generally like states to be "positive", cuz it makes thinking through the logic easier ("is game creation allowed?" vs "is game creation not disallowed?"), but it's used in very few places so I think you're correct here. I'll make the change.